### DISCUSSION TITLE
i'm given two zip files, and i don't know which one to extract.

### DISCUSSION THREAD

**Discussion by TallmadgeInsanityAnimations** [Maerciez/TheMillionaireGame#74] [Q&A] [answered]
![Screenshot 2024-03-23 085117](https://github.com/Macronair/TheMillionaireGame/assets/74796135/6c039da2-64b4-42dd-8e13-eac3526e2f07)

**Comment by Macronair** [comment, marked as answer]
Sorry to answer so late from my side.
But you probably need the 'MillionaireGame_v1.1.756' zip file. Otherwise you can download the right zip file again from the releases page where there are also newer versions available.
